net/snode.c/Config.in

```diff
@@ -0,0 +1,344 @@
+menu "Configuration"
+	depends on PACKAGE_snode.c-utils
+
+	config SNODEC_GROUP_ID
+		int
+		prompt "Group id of group snodec"
+		default "1023"
+		help
+				The group id for unix group 'snodec'. This group is used for config/log/pid file
+				management.
+				(default: 1023)
+
+				https://github.com/VolkerChristian/snode.c
+				Volker Christian <me@vchrist.at>
```

Comment:
Why do we need to have this? This is here multiple times. What is the reason?

Reply:
There was no real reason doing this. I just remember I saw this in an others package makefile - but I don't remember the package.
There is no need for this and I have removed it like you suggested.